backend/BackendMIIT/src/main/java/com/example/BackendMIIT/models/domain/Direction.java

```diff
@@ -3,13 +3,17 @@
 import jakarta.persistence.Entity;
 import jakarta.persistence.OneToMany;
 import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.util.List;
 
 @Entity
 @Table(name = "directions")
 @Data
+@AllArgsConstructor

```

Comment:
RequiredArgsConstructor, which included in Data annotation is enough, so you can remove it

Reply:
Removed